README.md

```diff
@@ -1,20 +1,25 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
 # LeaderBoard
+
```

Comment:
- [ ] You are almost there! However, as the last reviewer said, your README file is missing some important parts like the setup instructions, install instructions, usage instructions and any other relevant information for the user who is going to test your app. I would recommend using the provided [template](https://github.com/microverseinc/readme-template) from Microverse.